lib/ansible/module_utils/vmware.py

```diff
@@ -133,7 +133,7 @@ def connect_to_api(module, disconnect_atexit=True):
     except vim.fault.InvalidLogin, invalid_login:
         module.fail_json(msg=invalid_login.msg, apierror=str(invalid_login))
     except requests.ConnectionError, connection_error:
-        if '[SSL: CERTIFICATE_VERIFY_FAILED]' in str(connection_error) and not validate_certs:
+        if 'certificate verify failed' in str(connection_error) and not validate_certs:
```

Comment:
I'm going to guess this string has changed across versions of the product, I would try both just to keep it backwards compatible